jobs/async-upload/samples/sample_job_s3_to_oci.yaml

```diff
@@ -17,7 +17,7 @@ metadata:
 type: kubernetes.io/dockerconfigjson
 stringData:
   .dockerconfigjson: '{"auths": {"distribution-registry-test-service.default.svc.cluster.local:5001": {"auth": "base64(username:password)","email": "user@example.com"}}}'
-  ACCESS_TYPE: ["Push", "Pull"]
+  ACCESS_TYPE: "Push,Pull"
```

Comment:
I'm not 100% sure if this will be a comma-separated list or a stringified array. in either case, an array-type is not valid for a secret's value